frontend/benefit/handler/src/pages/application/index.tsx

```diff
@@ -0,0 +1,32 @@
+import ApplicationReview from 'benefit/handler/components/applicationReview/ApplicationReview';
+import AppContext from 'benefit/handler/context/AppContext';
+import { GetStaticProps, NextPage } from 'next';
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations';
+import * as React from 'react';
+import { useEffect } from 'react';
+import theme from 'shared/styles/theme';
+
+const ApplicantIndex: NextPage = () => {
+  const { setIsNavigationVisible, setLayoutBackgroundColor } =
+    React.useContext(AppContext);
+
```

Comment:
Perhaps this could be cleaner code?
```
const setPageSettings = React.useCallback(() => {
    setIsNavigationVisible(false);
    setLayoutBackgroundColor(theme.colors.white);
}, [setIsNavigationVisible, setLayoutBackgroundColor]);

useEffect(() => {
    setPageSettings();
    return () => setPageSettings();
}, [setPageSettings]);
```

Reply:
lets see how many configurations we have. maybe we can do the pageConfigs hook or something. its a first try